src/sage/categories/groups.py

```diff
@@ -453,51 +453,26 @@ def conjugacy_class(self, g):
 
         def minimum_generating_set(self):
             """
-            Return a list of the minimum generating set of this group.
+            Returns a list of the minimum generating set of this group.
 
             EXAMPLES::
 
-                sage: G = GL(2,GF(3))
-                sage: G.minimum_generating_set()
-                [
-                [1 2]  [1 2]
-                [0 1], [1 1]
-                ]
-
-                sage: G = SymmetricGroup(3)
-                sage: s = G.minimum_generating_set(); s
-                [(2,3), (1,2,3)]
-                sage: s[0].parent()
-                Symmetric group of order 3! as a permutation group
-
-                sage: A5 = AlternatingGroup(5)
-                sage: A5.minimum_generating_set()
-                [(1,2,3,4,5), (3,4,5)]
-
-                sage: H = groups.matrix.Heisenberg(1,3); H
-                Heisenberg group of degree 1 over Ring of integers modulo 3
-                sage: H.minimum_generating_set()
-                [
-                [1 1 0]  [1 0 0]
-                [0 1 0]  [0 1 1]
-                [0 0 1], [0 0 1]
-                ]
-
-            TESTS:
-
-            Test that function gives an error for infinite groups::
-
-                sage: G = GL(2, ZZ)
-                sage: G.minimum_generating_set()
-                Traceback (most recent call last):
-                ...
-                NotImplementedError: only implemented for finite groups
+                sage: def A_5_to_n(n):
+                ....:     A5 = AlternatingGroup(5).gap()
```

Comment:
Do not change anything except algorithm section in the docstring because this is also not in proper format which sagemath uses.